validator/accounts/v2/cmd.go

```diff
@@ -0,0 +1,30 @@
+package v2
+
+import (
+	"github.com/urfave/cli/v2"
+
+	"github.com/prysmaticlabs/prysm/shared/featureconfig"
+	"github.com/prysmaticlabs/prysm/validator/flags"
+)
+
+// Commands for accounts-v2 for Prysm validator clients.
+var Commands = &cli.Command{
+	Name:     "accounts-v2",
+	Category: "accounts-v2",
+	Usage:    "defines commands for interacting with eth2 validator accounts (work in progress)",
+	Subcommands: []*cli.Command{
+		{
+			Name: "new",
+			Description: `creates a new validator account for eth2. creates a new wallet for a user based on
+specified input, capable of creating a direct, derived, or remote wallet.
+this command outputs a deposit data string which can be used to deposit Ether into the ETH1.0 deposit
+contract in order to activate the validator client`,
```

Comment:
Is this comment accurate? Should it be `in order to create a validator account`?